<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1091</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/121113.png</normaloff>../../../Downloads/121113.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
			background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;

	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/1170718.png</normaloff>../../../Downloads/1170718.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
			background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 18px;
	
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Pdf</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/PDF_file_icon.svg.png</normaloff>../../../Downloads/PDF_file_icon.svg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="mail">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>130</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	
	
		background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>mail</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/275722247_3127640084221767_4987571222713071970_n.png</normaloff>../../../Downloads/275722247_3127640084221767_4987571222713071970_n.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="calendrier">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;

	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}
</string>
      </property>
      <property name="text">
       <string>calendrier</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/32223.png</normaloff>../../../Downloads/32223.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;

	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/271743198_464449928646161_2550356874147298555_n.png</normaloff>../../../Downloads/271743198_464449928646161_2550356874147298555_n.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;etat &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>681</width>
        <height>281</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajout">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(184, 247, 213);
	
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/64579.png</normaloff>../../../Downloads/64579.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modify">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		background-color: rgb(184, 247, 213);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}


QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/45706.png</normaloff>../../../Downloads/45706.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}
</string>
      </property>
      <property name="text">
       <string>confirmer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_pdf">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);
  color: black;
  padding: 10px;
  text-align: center;
font-family:calibri;
  font-size: 13px;
  margin: 2px 1px;
  border-radius : 16px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_etat">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
      <property name="placeholderText">
       <string>etat batterie....</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);
font: 9pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>your id please....</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>20</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}</string>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>391</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 247, 213);</string>
      </property>
      <property name="placeholderText">
       <string>your text.........</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}</string>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1235</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
